### DISCUSSION TITLE
Example AP profile request

### DISCUSSION THREAD

**Discussion by shartcarbuncle** [nguyenquyhy/Flight-Tracker-StreamDeck#235] [Q&A] [answered]
Hi gang,

This add-on seems really nifty. I got a Stream Deck (standard size) just for it. I probably should have done a bit more research, because I didn't realize how much initial setup there would be. I know I'll figure it out eventually, but I was wondering if I could download a profile along the lines of the one in the example screenshot I'm attaching to this post. It would save me a lot of time to have a starting point like that, assuming it exists. If it was just a mockup image to show would you _could_ make, then I'll just have to slog it out myself. :)

I looked around in the sharing thread and didn't see anything like it; I wasn't sure if I should post this request in there or separately. 

Thanks!

![AP Profile Example](https://user-images.githubusercontent.com/80048952/114614739-e5c48b00-9c72-11eb-8d54-ffa836d9c74b.png)

**Comment by nguyenquyhy** [comment]
It's actually my profile. I'll export and share later.

Also, I'm working on a better way to help everyone to get started with pre-built profiles.

**Comment by shartcarbuncle** [comment]
Thanks so much, that’s brilliant! That particular layout just looks perfect for the way I fly.

**Comment by phenDruck** [comment]
Boy that REALLY would be helpful. Having an example is huge best teaching tool. Thanks for all your 😓 hard work!!!

"Democracy is two wolves and a lamb voting on what to have for lunch. Liberty is a well-armed lamb contesting the vote." -Ben Franklin

**Comment by nguyenquyhy** [comment, marked as answer]
Here you go. Please unzip it and double click the file.

[Flight Tracker.zip](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/files/6307958/Flight.Tracker.zip)

**Reply by shartcarbuncle** [reply]
Hugely appreciated, thanks.

**Comment by phenDruck** [comment]
Thanks so much!

"Democracy is two wolves and a lamb voting on what to have for lunch. Liberty is a well-armed lamb contesting the vote." -Ben Franklin
